AUTHORS

```diff
@@ -3,12 +3,14 @@ Asuki Kono
 belono
 Christoph Heuel
 Cody (Quantified Code Bot)
+csoft2k
 Curtis // mashedkeyboard
 Davis Goglin
 Dean Rispin
 Dmytro Katyukha
 Hark
 Joel Lehtonen
+K
```

Comment:
This is a [feature of git-shortlog](https://git-scm.com/docs/git-shortlog#_mapping_authors). You just have to add your name-mappings to the .mailmap-file in the root directory of this project.

So you could just add the line `csoft2k <csoft2k@hotmail.com>`, which will replace the name "K" in the one commit that you made, when creating shortlogs and similar stuff. ;-)
Otherwise, this is something that I can also fix, if you want.